sevntu-checks/sevntu-checks.xml

```diff
@@ -216,6 +216,14 @@
       <property name="ignoreMethod" value="false"/>
     </module>
 
+    <!-- Jsr305 -->
```

Comment:
Move this to the rest of the coding package.

Reply:
Done.